website/src/doc-demos/aggregation-layers.js

```diff
@@ -12,9 +12,10 @@ import makeLayerDemo from './layer-demo';
 import {DATA_URI} from '../constants/defaults';
 
 export const ContourLayerDemo = makeLayerDemo({
-  getTooltip: ({object}) => `threshold: ${object.contour.threshold}`,
-  layer: new ContourLayer({
-    data: `${DATA_URI}/sf-bike-parking.json`,
+  Layer: ContourLayer,
+  getTooltip: '({object}) => object && `threshold: ${object.contour.threshold}`',
+  props: `{
```

Comment:
`props` is now a string - this is necessary for populating the Codepen source. Without quoting it here all callbacks will be transpiled and uglified during build.

Reply:
Hmm... that does not seem ideal, but I guess this code was never intended for end users.